libcaf_core/tests/legacy/typed_spawn.cpp

```diff
@@ -5,16 +5,16 @@
 #include "caf/config.hpp"
 
 // exclude this suite; seems to be too much to swallow for MSVC
-#ifndef CAF_WINDOWS
+// #ifndef CAF_WINDOWS
```

Comment:
If MSVC is fine with the suite, just remove the lines instead of commenting them out.